tables/DataMan/VirtualTaQLColumn.cc

```diff
@@ -157,7 +157,7 @@ void VirtualTaQLColumn::prepare()
   TableColumn tabcol (table(), itsColumnName);
   itsExpr = tabcol.keywordSet().asString ("_VirtualTaQLEngine_CalcExpr");
   // Compile the expression.
-  TaQLResult res = tableCommand ("calc from $1 calc " + itsExpr, table());
+  TaQLResult res = tableCommand ("calc " + itsExpr + " from $1" , table());
```

Comment:
You are right. That change accidently became part of the change set. Very
sloppy of me. I'll remove it.
Thanks.

On Wed, May 25, 2016 at 5:15 PM, Julian Taylor <notifications@github.com>
wrote:

> In tables/DataMan/VirtualTaQLColumn.cc
> <https://github.com/casacore/casacore/pull/426#discussion_r64591472>:
>
> > @@ -157,7 +157,7 @@ void VirtualTaQLColumn::prepare()
> >    TableColumn tabcol (table(), itsColumnName);
> >    itsExpr = tabcol.keywordSet().asString ("_VirtualTaQLEngine_CalcExpr");
> >    // Compile the expression.
> > -  TaQLResult res = tableCommand ("calc from $1 calc " + itsExpr, table());
> > +  TaQLResult res = tableCommand ("calc " + itsExpr + " from $1" , table());
>
> can you explain how this relates to the column cloning?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/casacore/casacore/pull/426/files/d5bb137b629a65509d34ac135508cc66bf353dd8#r64591472>
>